src/Assets/acuteConditionOptions.tsx

```diff
@@ -9,6 +9,55 @@ import { ReactComponent as Job_resources } from './OptionCardIcons/AcuteConditio
 import { ReactComponent as Legal_services } from './OptionCardIcons/AcuteConditions/legal_services.svg';
 import { ReactComponent as Support } from './OptionCardIcons/AcuteConditions/support.svg';
 
+export const acuteConditionResultMapping = {
```

Comment:
Because the urgent needs have the type text already, I think that we can just make this an icon map. Also, I think we should use a different class name because this is not being used in an option card.

Example: 
```
{
  'Food or groceries': <Food className="urgent-need-icon" />,
  ...
}
```

Reply:
The issue is that the type text is too long to use as headers. I created `acuteConditionResultMapping` to hold the heading text and `const translateNeed` to pass said text to `ResultTranslate`

Reply:
I think that we should use the type text from the API, and then we can shorten the type text on the urgent need type.

@kbottagaro Can we update the urgent need types when we release this change.

Reply:
Ok, I'll change it to use the type text from the API. In order to map it to the corresponding icon, I believe `acuteConditionResultMapping` will still need to hold the icon and the new shortened type text.

Reply:
The type text can be changed in the database.

If you go to http://localhost:8000/api/translations/admin/urgent_needs
You can then select whichever program you want, and then select the type link, and edit the type text.

Reply:
> The type text can be changed in the database.
> 
> If you go to http://localhost:8000/api/translations/admin/urgent_needs You can then select whichever program you want, and then select the type link, and edit the type text.

I updated the backend type texts locally. See screenshots below. 

![Screenshot 2024-02-13 at 2 48 09 PM](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/27e00955-c89e-418f-82e7-d452bd427ac4)

![Screenshot 2024-02-13 at 2 43 23 PM](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/1b7c60bd-fb73-4bb1-bb97-f3eae94c2e4a)